### PR TITLE
add elbv2 probe and fix minor issues

### PR DESCRIPTION
**Pull request by 0rc**
Signed-off-by: Dmitry Batiievskyi <dmitriy@batiyevsky.org>

add elbv2 probe to check if all targets are healthy

Fix some issues in tests:
* unused imports
* test name duplicates
* unused variables
* formatting (spaces, indentation etc)
Not fixed:
* line length

### CODE REVIEW

**Comment by 0rc** [comment]
I don't think it's worth mentioning this in changelog

File: tests/elbv2/test_elbv2_probes.py [hunk 1]
```diff
@@ -47,3 +49,50 @@ def test_targets_health_count(aws_client):
     response = targets_health_count(tg_names=tg_names)
     assert response['TestTargetGroup1']['healthy'] == desired_response['TestTargetGroup1']['healthy']
     assert response['TestTargetGroup2']['unhealthy'] == desired_response['TestTargetGroup2']['unhealthy']
+
+
+@patch('chaosaws.elbv2.probes.aws_client', autospec=True)
+def test_all_targets_healthy(aws_client):
```
Comment (russmiles) [inline tests/elbv2/test_elbv2_probes.py]: Should we have a test for a positive result also? This test ensures we get the right response of "false" for the mock data, but perhaps one also to show that true is expected when all are healthy?

**Comment by russmiles** [comment]
Would also like this in the changelog, it's a new probe so is a new feature?

File: tests/elbv2/test_elbv2_probes.py (hunk 1 above)
Comment (0rc) [reply tests/elbv2/test_elbv2_probes.py]: no problems, adding

**Comment by 0rc** [comment]
@russmiles we already have this in changelog under unreleased:
`support for elbv2 with basic probes and action`

**Comment by 0rc** [comment]
the ordering issue you tried to fix with https://github.com/chaostoolkit-incubator/chaostoolkit-aws/commit/3a00214990580bc40ad16a8b2394c8adcf182cb8 is not fixed...

**Comment by 0rc** [comment]
'true' test added, ordering issue fixed

**Comment by russmiles** [comment]
Want me to trigger a release @0rc with this probe added, of 0.7.0, or are there any further elb features to be added before that release?

**Comment by 0rc** [comment]
no I am not planning more elb features soon, but will be adding asg probes and actions shortly

**Comment by russmiles** [comment]
In which case let's wait for those features and we can roll _that_ out as 0.7.0. Great stuff @0rc , thanks so much for your efforts on this project!

**Pull request merged**